frontend/src/components/horizontalPodAutoscaler/List.tsx

```diff
@@ -40,7 +40,7 @@ export default function HpaList() {
                 if (hpa.metrics.length > 1) {
                   value.push(
                     <StatusLabel key="more..." status="">
-                      {metrics.length - 1} {t('frequent|more...')}
+                      {metrics.length - 1} {t('frequent|more…')}
```

Comment:
I guess the translation files need to be updated because of this.

Reply:
They are now.